packages/core/logger/src/index.js

```diff
@@ -33,137 +31,98 @@ export class Logger {
       }));
     };
     this.logFile = options.logFile;
-    this.context = options.context;
   }
 
-  /**
-   * Parses the logFile, returning an array of JSON objects containing the
-   * log messages.
-   * @param {Number} limit specifies how many log messages to return from the
-   *  end of the log file
-   * @returns {Array} array containing
-   *  - msg: the log message
-   *  - logLevel: log level (ie 'info', 'debug')
-   *  - name: process name (always "embark")
-   *  - timestamp: timestamp of log message (milliseconds since 1/1/1970)
-   */
-  parseLogFile(limit) {
-    let matches;
-    let logs = [];
-    const logFile = fs.readFileSync(this.logFile, 'utf8');
-    while ((matches = LOG_REGEX.exec(logFile)) !== null) {
-     // This is necessary to avoid infinite loops with zero-width matches
-     if (matches.index === LOG_REGEX.lastIndex) {
-       LOG_REGEX.lastIndex++;
-     }
-
-     if (matches && matches.length) {
-       logs.push({
-         msg: [matches[3]],
-         logLevel: matches[2],
-         name: 'embark',
-         timestamp: date.parse(matches[1], DATE_FORMAT).getTime()
-       });
-     }
-    }
 
-    // if 'limit' is specified, get log lines from the end of the log file
-    if(limit && limit > 0 && logs.length > limit){
-      logs.slice(limit * -1);
-    }
-    return logs;
+  registerAPICall(plugins) {
+    let plugin = plugins.createPlugin('dashboard', {});
```

Comment:
That's a good question. I **assume** this is actually a mistake. Looking for `createPlugin('dashboard')` inside our codebase shows that there are only two places - here, and in the dashboard module. 

It most likely should be `logger` instead. Although, OTOH, the logger isn't actually a plugin, so semantically that might not be correct.

Reply:
I've just noticed that there's a `registerAPICall()` on the `embark` object as well, so we might be able to drop the `plugin` altogether here.

Reply:
I stand corrected.

`registerAPICall()` is actually not an API on `embark`. It only looks like because we call the thing injected into plugin module `embark`. In reality, those are instances of `Plugin`, which is most likely the reason why we use `createPlugin()` here. It's the only way to get hold of a single `plugin` instance. 

Will change it to `logger` at least.